docs/src/development.md

```diff
@@ -1,16 +1,62 @@
 # Development
-The `License Manager` application incorporates a mix of different services in docker and LXD containers.
-This text will attempt to define the procedure for initializing and running the different components.
+The `License Manager` can be provisioned using two approaches:
 
-## Pre-Installation
+* Use `Docker Compose` to run the services
+* Create a `Slurm` cluster using Juju
```

Comment:
```suggestion
* Create a `Slurm` cluster and License Manager components using Juju.
```

You may need to note that all of this can be done _manually_ as well, but that it isn't covered by the guide.

Reply:
Updated it.